INSTALL.md

```diff
@@ -195,7 +196,7 @@ These commands build and install KIELE:
 git clone https://github.com/runtimeverification/iele-semantics.git
 cd iele-semantics
 git submodule update --init --recursive
-k-configure-opam
+opam init --yes
```

Comment:
these are interactive instructions, so we should not pass --yes in case users wish to configure their environment differently than the default.

Reply:
thanks for pointing that out